qemu/tests/cfg/qemu_guest_agent.cfg

```
@@ -531,6 +531,12 @@
             cmd_get_hostkey = "cat ~/.ssh/id_rsa.pub"
             cmd_del_key_file = "rm -rf ${guest_homepath}/.ssh/authorized_keys"
             test_login_guest = ssh ${guest_user}@%s -o stricthostkeychecking=no ls /home
+        - gagent_run_qga_as_program:
+            only Windows
+            gagent_check_type = run_qga_as_program
+            enter_qemu_ga_program_path = cd "C:\Program Files\Qemu-ga\"
+            run_qga_program_cmd = start /b qemu-ga.exe 
```

```suggestion
            enter_qemu_ga_program_path = 'cd "C:\Program Files\Qemu-ga\" &'
            enter_qemu_ga_program_path += 'start /b qemu-ga.exe'
```
would be more efficienct.